SofascoreAcademy/SofascoreAcademy/Views/MainViewController.swift

```diff
@@ -0,0 +1,166 @@
+//
+//  MainViewController.swift
+//  SofascoreAcademy
+//
+//  Created by Akademija on 08.04.2024..
+//
+
+import Foundation
+import UIKit
+import SofaAcademic
+import SnapKit
+
+protocol parentSportSlugPicker {
```

Comment:
```suggestion
protocol ParentSportSlugPicker {
```

Ovo bi sve trebalo pocinjati velikim slovom

Reply:
Ali opcenito ove protokole mozes izdvojiti svaki u svoj file.